docs/src/main/asciidoc/azure-functions-http-guide.adoc

```diff
@@ -0,0 +1,120 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+
+include::./attributes.adoc[]
+= {project-name} - Using Resteasy, Undertow, or Vertx Web with Azure Functions (Serverless)
```

Comment:
use `Quarkus` instead of `{project-name}`

Reply:
It's a bit more than that: the title needs to be just after the end of the comment and the includes after the title. Otherwise, the website doesn't take it. Thus the variable cannot be used as it's declared after the title itself.